### DISCUSSION TITLE
Importing SharpMap into Unity

### DISCUSSION THREAD

**Discussion by Marisakis** [SharpMap/SharpMap#208] [Q&A]
Hey there,

I'm trying to find the most practical way of connecting to a Geoserver in Unity. For that I've tried to import SharpMap into the Unity editor as a DLL, but I'm running into dependency problems.
As dependencies I'd need the GeoAPI, Common.Logging and NetTopologySuite. But these, as far as I can tell, are all made for the Windows platform, requiring additional dependencies such as System.Drawing, System.Memory and System.Buffer. 

Is importing this at all viable, or would I be better off just trying to copy as much of the WFSclient and Shapefile and making my own implementation, manually?

**Comment by FObermaier** [comment]
GeoAPI, Common.Logging and NetTopologySuite are not made __just__ for windows.
System.Drawing, System.Memory and System.Buffer are available for different platforms as well.

I have no clue about the requirements of programming for unity, sorry
